templates/signup.pug

```diff
@@ -6,8 +6,6 @@ block content
     | Create an account or
     = ' '
     a(href='/sessions/new') login
-    
-  p= error
```

Comment:
This removes functionality I think? Now the user has no way to see what validation errors occurred?